.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.analysis.typeCheckingMode": "off"
```

Comment:
Lost this comment by making the review :/

I've ended up with the following in my settings.json file. I'm not sure why or if I need to keep it, but given that these .vscode files don't merge nicely with user and repository configurations, do we need to chat about what we use them for?

```json
{
  "python.testing.pytestArgs": [
    "tests"
  ],
  "python.testing.unittestEnabled": false,
  "python.testing.pytestEnabled": true
}
```

Reply:
I'm not sure how this snuck into the PR, I didn't add this on purpose